src/components/CheckBox.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { View, Text, Image, StyleSheet } from 'react-native';
+import { RectButton } from 'react-native-gesture-handler';
+
+import { icons, theme } from '../constants';
+
+const { FONTS, COLORS } = theme;
+const BOX_SIZE = 11;
```

Comment:
Same about indentation and ESLint, comments about that I left in MainStackNavigator.js file.